### DISCUSSION TITLE
PM Questions

### DISCUSSION THREAD

**Discussion by sanya301** [hackforla/HomeUniteUs#693] [Q&A]
Place for PMs to ask questions to other PMs outside of specific stories/epics

**Comment by lasryariel** [comment]
Can you please clarify how we are meant to engage with Engineers on this project?

- What is the current structure of the engineering team, particularly who are we communicating with primarily for given engineering responsibilities?
- What are PMs responsibilities to the Engineers?
- Where can we learn about the engineering that has already gone into the application, and how can we make sure our stories are considerate of these details?

**Comment by lasryariel** [comment]
Are you all aware of this doc? Should we be maintaining something like it? [HUU Master User Stories](HUU Master User Stories)

**Comment by lasryariel** [comment]
What is the best place to notify the other PMs that I have a story written in Google Docs which is ready for feedback? Is it best to just post in the #home-unite-us-pm slack channel?
